pkg/sso/sso.go

```diff
@@ -52,6 +53,13 @@ type auth interface {
 	CheckLogin(kthid, id string) error
 }
 
+type ssoUser struct {
+	Email      string `json:"email"`
+	FirstName  string `json:"firstName"`
+	FamilyName string `json:"familyName"`
+	YearTag    string `json:"yearTag"`
```

Comment:
```suggestion
	Email      string `json:"email,omitempty"`
	FirstName  string `json:"firstName,omitempty"`
	FamilyName string `json:"familyName,omitempty"`
	YearTag    string `json:"yearTag,omitempty"`
```

This will probably not make a big difference, but for users without a year tag, it can make a difference depending on how the application checks for the tag's existence.